### ISSUE TITLE
feedback on [ML on GCP C6] Training on Cloud ML Engine lab

### ISSUE THREAD

**Issue by tarrade** [GoogleCloudPlatform/training-data-analyst#399]
Nice tutorial, thanks. Some feedback from a user point of view:

datalab > notebooks > training-data-analyst > courses > machine_learning > deepdive > 06_structured >5_train.ipynb.

- adding export CLOUDSDK_PYTHON=/usr/local/envs/py3env/bin/python allow to make local prediction
%bash
export CLOUDSDK_PYTHON=/usr/local/envs/py3env/bin/python
MODEL_LOCATION=$(ls -d $(pwd)/babyweight_trained/export/exporter/* | tail -1)
echo $MODEL_LOCATION
gcloud ml-engine local predict --model-dir=$MODEL_LOCATION --json-instances=inputs.json

-TensorBoard
it seems the number of  "points" to display is incorrect since only one point is display for all metrics (loss ...)

- As mentioned in the lab it takes between 50 min to 1h40 to run the jobs. It is quite challending to run the first training job, then run the one from hyperparameter tuning. Wait for the results to submit the final job with the best parameter from the hyperparameter tuning job.

- I don't know if this is a temporary glitch or an issue with permission but it is not possibleto view the job of the ML engine.

Thanks again

**Comment by tarrade** [comment]
With the new version of Tensorflow 2.0, Tensorbaord running in notebook, the new notebook instance int=stead of datalab everything willneed to be updated soon and my comment are then obsolete. Looking forward the updated version with all the changes mentioned above. Thanks.
